### ISSUE TITLE
sync Airtable with ArcGIS

### ISSUE THREAD

**Issue by nohorse** [OpenEugene/fire-refuge#19]
As part of the build process read the AirTable and create a static CSV file.
Host that file as a static resource here  http://FireRefuge.com/data/Shelter.csv
Then @thomasfiorelli  can point to that for the his layer data.

**Comment by nohorse** [comment]
@thomasfiorelli  try this: https://raw.githubusercontent.com/OpenEugene/fire-refuge/master/static/data/shelter.csv

**Comment by nohorse** [comment]
I'm looking into airtable apps

**Comment by nohorse** [comment]
@thomasfiorelli when you are setting up the URL for CSV download can you set the interval.  like every X hours?

**Comment by thomasfiorelli** [comment]
yes, that is correct. Once I add the the CSV to the ESRI server on my end, I have the ability to change the refresh interval directly from the Web Map. Once changed, that setting will carry forward to the Web App we see in the online link.

**Comment by nohorse** [comment]
@thomasfiorelli I'll have a service up and running today.  please verify that you can consume this style of csv (attached)
[response.zip](https://github.com/OpenEugene/fire-refuge/files/5240187/response.zip)

**Comment by thomasfiorelli** [comment]
The .csv imports properly and the coordinates are displaying on the map.

**Comment by nohorse** [comment]
here's a valid service link you can try hosted on the Open Eugene CfA Azure: https://cachetable.azurewebsites.net/csv/FireRefuge/Shelters  

I'll move this to MVP studio soon.  Please test that link and let me know if it works.

**Comment by thomasfiorelli** [comment]
While the data imports, I am warned of the following: 
![image](https://user-images.githubusercontent.com/55812658/93513569-e498ce80-f8da-11ea-8485-a442cf9f8e34.png)

**Comment by nohorse** [comment]
might be the blank row.

**Comment by thomasfiorelli** [comment]
This warning may be trivial. Looking through the attribute table on the GIS web map, I can see there are 8 attributes, which appears to match the number of records in the .csv. All fields appear to be showing.

**Comment by nohorse** [comment]
this is great news!  I'll show you how to configure other tables and views.  Here's some quick drawings:
![image](https://user-images.githubusercontent.com/311063/93514077-99cb8680-f8db-11ea-9928-07cfb6b35812.png)
![image](https://user-images.githubusercontent.com/311063/93514331-ef079800-f8db-11ea-9efe-5d012058b862.png)
![image](https://user-images.githubusercontent.com/311063/93514953-ae5c4e80-f8dc-11ea-8255-048fa0da8c41.png)

**Comment by thomasfiorelli** [comment]
Thank you for the run down! The Airtable API and the ability to customize and dispaly different views for different purposes from the same dataset are incredilbly useful.
